### PR TITLE
DigiTrust Facade init GH Issue 3911

### PR DESCRIPTION
**Pull request by goosemanjack**
Fix to init the DigiTrust facade object if the userId framework would not normally execute the call due to an ID being found.

## Type of change

- [X ] Bugfix

## Description of change
This executes code to build the DigiTrust facade after a time if it has not already been built.

- [X ] official adapter submission

## Other information
Fixes issue 3911

### CODE REVIEW

**Comment by mkendall07** [comment]
This pull request **introduces 1 alert** when merging 1185a0207abcbce4ac82e2a885163917bd99c8c6 into 842cc19f712c644db84aae10d7d3567514d481e8 - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-0228671c8ddcce3f7d0b863aaae2c2eb953cbdac)

**new alerts:**

* 1 for Superfluous trailing arguments

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by idettman** [comment]
Can you add a unit test for this scenario?

**Comment by goosemanjack** [comment]
I have added in a unit test per request.

**Comment by mkendall07** [comment]
This pull request **introduces 1 alert** when merging c9713db4dad4bfe959dcd6a687491409211f56a3 into 2bd04a1f9b7f706d9c60d8b1f401d9b743a8c8a3 - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-0a02fc65a358f2578c8391f875471e3938893503)

**new alerts:**

* 1 for Superfluous trailing arguments

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by goosemanjack** [comment]
Updated conflicts and merged changes from master. The failed test is a timeout in Safari and appears to be an unrelated intermittent issue.

File: modules/digiTrustIdSystem.js [hunk 1]
```diff
@@ -336,4 +347,19 @@ export const digiTrustIdSubmodule = {
...
+      member: 'fallback',
+      callback: noop
+    };
+    getDigiTrustId(conf);
+  }
+}
+
+fallbackTimer = setTimeout(fallbackInit, fallbackTimeout);
+
+attachIdSystem(digiTrustIdSubmodule);
```
Comment (idettman) [inline modules/digiTrustIdSystem.js]: delete this line (attachIdSystem was replaced by submodule)

File: modules/digiTrustIdSystem.js [hunk 2]
```diff
@@ -9,11 +9,13 @@
  * @requires module:modules/userId
  */
 
-// import { config } from 'src/config';
 import * as utils from '../src/utils'
 import { ajax } from '../src/ajax';
 import { submodule } from '../src/hook';
-// import { getGlobal } from 'src/prebidGlobal';
+import { attachIdSystem } from '../modules/userId/index';
```
Comment (idettman) [inline modules/digiTrustIdSystem.js]: delete the import for attachIdSystem (attachIdSystem was replaced by submodule)

Comment (goosemanjack) [reply modules/digiTrustIdSystem.js]: I have made the indicated changes. Please review and LMK if anything else is needed.

**Pull request merged**